src/packages/@ncigdc/modern_components/Lolliplot/separateOverlapping.js

```diff
@@ -0,0 +1,21 @@
+export const overlaps = (a, b) => {
+  const first = a.start < b.start ? a : b;
+  const last = first.start === a.start ? b : a;
+  return first.end > last.start;
+};
+
+const separateOverlapping = data => {
+  return data.reduce(
+    (acc, d) => {
+      const index = acc.findIndex(level => level.every(l => !overlaps(l, d)));
+
+      if (index > -1) acc[index].push(d);
+      else acc.push([d]);
+
+      return acc;
+    },
+    [[]],
```

Comment:
You shouldn't need the empty array in the array :D with `[]` index will equal `-1` so it will push a new array. 
not sure if it matters, but this way if `data === []`  it will return `[[]]` instead of `[]`. `[]` seems more correct? 
(sorry for the nitpicking :P )